build.sbt

```diff
@@ -2,12 +2,19 @@ name := "dfdl-envelope-payload"
 
 organization := "io.github.dfdlschemas"
 
-version := "1.0.0"
+version := "1.1.0"
+
+daffodilVersion := "3.9.0"
 
 libraryDependencies ++= Seq(
-  "io.github.dfdlschemas" % "dfdl-tcpmessage" % "1.0.0",
+  "io.github.dfdlschemas" % "dfdl-tcpmessage" % "1.1.0",
   "com.owlcyberdefense" % "dfdl-mil-std-2045" % "1.3.2",
-  "com.tresys" % "dfdl-pcap" % "1.4.0",
+  "com.tresys" % "dfdl-pcap" % "1.4.0"
 )
 
 enablePlugins(DaffodilPlugin)
+daffodilPackageBinVersions := Seq(daffodilVersion.value)
+daffodilPackageBinInfos := Seq(
+  ("/io/github/dfdlschemas/envelopepayload/xsd/envelopePayload.dfdl.xsd", Some("message"), None)
```

Comment:
Doesn't the daffodil plugin use DaffodilBinInfos now? I wasn't aware this way still worked

Reply:
Yeah, fixing this and also adding a config file to suppress compiler warnings that comes from PCAP's files.